spec/calendar.html

```diff
@@ -54,761 +49,715 @@ <h1>
   <emu-clause id="sec-temporal-calendar-abstract-ops">
     <h1>Abstract Operations for Temporal.Calendar Objects</h1>
```

Comment:
Good catch. Additionally, grepping for further occurences of Temporal.Calendar and Temporal.TimeZone revealed quite a lot more. I've cleaned these out but kept them separate in a fixup commit for the time being, in case anyone who already looked at the docs in this PR wants to see the differences.